### ISSUE TITLE
Newer PiDeck Build

### ISSUE THREAD

**Issue by MrShigura** [pideck/pideck#81]
I received an email with the newer build of pideck back in may, but missed the email at the time and the link is now dead. Would appreciate it if you could re-send the newer build?

**Comment by danielhjames** [comment]
Hi @MrShigura please try the 0.2 test image at https://github.com/pideck/pideck-distro/releases and let me know how you get on.

**Comment by MrShigura** [comment]
Thanks, much appreciated, will be trying it out later today

**Comment by MrShigura** [comment]
> Hi @MrShigura please try the 0.2 test image at https://github.com/pideck/pideck-distro/releases and let me know how you get on.

So far so good, however, using pideck with the official 7" touchscreen, the orientation is upside down at load, what's the easiest way to change this?

**Comment by GuusR** [comment]
This might help: https://github.com/pideck/pideck/issues/63

**Comment by danielhjames** [comment]
Hi @MrShigura could it be you just wired your touchscreen the different way up to us? Do you have a twist in the ribbon cable?

**Comment by MrShigura** [comment]
> Hi @MrShigura could it be you just wired your touchscreen the different way up to us? Do you have a twist in the ribbon cable?

It's possible, the orientation I'm using currently is required for the ports to be accessible however.

**Comment by kosmopoisk** [comment]
tell me, I understand that PIDVS is the continuation of the development of PiDeck, tell me where you can download PIDVS for familiarization and building the system?

**Comment by GuusR** [comment]
@kosmopoisk  PIDVS is a completely different project I believe, and as far as I know it is not (and maybe won't ever be) released nor is it open source. Not even sure if the guy is still working on it, last update on the project is from may 2017. https://samspreadborough.com/2017/05/21/pidvs-2-0-update/

**Comment by kosmopoisk** [comment]
Thanks, but where can I download this project?

**Comment by GuusR** [comment]
PiDeck is available for download here: https://github.com/pideck/pideck-distro/releases
PIDVS is not available for download anywhere I'm afraid

**Comment by kosmopoisk** [comment]
Tell me, is it possible to connect two Raspberry Pi over the network for synchronization?

**Comment by kosmopoisk** [comment]
Is it possible, two PiDeck, to connect with each other over the network?

**Comment by GuusR** [comment]
Nope, linking two pidecks is not possible. But I wouldn’t know what you would use it for, only thing I can imagine is not having to deal with two usb-sticks. Syncing wouldn’t be possible anyway, since the pitch is controlled by tt or cdj. If you want to control two decks this can be done with a single pideck. 
I suggest you do some reading on what the pideck can and can’t do and see if it’s something for you. Start with the website, github readme’s and issue pages and maybe check the YouTube channel including comment section. A lot of your questions are probably already answered there.

**Comment by kosmopoisk** [comment]
https://www.youtube.com/watch?v=0VQwE0gD2yE

look for what, how is synchronization done

**Comment by danielhjames** [comment]
Thanks @kosmopoisk that video shows software which is not released, as far as I know. 

Mounting the library across ethernet is possible of course. The other task I can forsee is that when the USB stick is unplugged from one deck, we need to send a message to the other deck to rescan the library there too. There would probably need to be some file watching daemon and also a message passing system taking care of this. The trick would be to find a lightweight solution.

**Comment by kosmopoisk** [comment]
I need the deck to catch the squares for the vault, and the track on the screen itself was beaten on the screen, plus add a sync button

**Comment by danielhjames** [comment]
Hi @kosmopoisk we have no plans to add a sync button, it's full manual control! Not sure what you mean by 'squares'.

**Comment by kosmopoisk** [comment]
![pidvs-v2-0-overview-screen](https://user-images.githubusercontent.com/43547203/52319937-57428800-29dd-11e9-843c-f9f4f84a519c.jpeg)
![default](https://user-images.githubusercontent.com/43547203/52319938-57428800-29dd-11e9-9952-cc9f88032e7a.png)

**Comment by kosmopoisk** [comment]
I'm sorry, I am writing through a translator, since I am from Russia

**Comment by kosmopoisk** [comment]
two decks connected by rj45 network
![1](https://user-images.githubusercontent.com/43547203/52331547-8838b200-2a09-11e9-9821-8da4cbc8e147.png)
![2](https://user-images.githubusercontent.com/43547203/52331548-8838b200-2a09-11e9-8312-278146249132.png)

**Comment by danielhjames** [comment]
Hi @kosmopoisk I see what you mean, these squares relate to 'beat grid' and the ability to loop on the beat at various beat lengths. In xwax we do not have this feature, but Mixxx does.

The RJ45 cable sync is a feature of PiDVS which is not in PiDeck. If someone is willing to work on this feature it would be very welcome :-)

**Comment by beaucasque** [comment]
Hey! I think that pideck is perfect. Cue points,  simple midi implementation and the use of a pi4 to benefice from those fast usb3 ports is the only thing that could be missing. For a gig, I often bring a big library. I need a way to browse fast. So using midi, i could assign a jog wheel to scroll thru long lists. Thank you so much. Amazing project.

**Comment by danielhjames** [comment]
Hi @beaucasque the simplest method for large library searches is to attach a small USB QWERTY keyboard. Text searches are supported in xwax, plus you have all the keyboard shortcuts such as changing the sort order: https://github.com/pideck/pideck/blob/master/keyboard_control.md

**Comment by beaucasque** [comment]
Would I be able to use a trackpad to scroll to my libraries. Also, is there a way to use folders on the USB drive and search inside each folder instead of the entire drive?

Once again, Thank you!

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by danielhjames** [comment]
Hi @beaucasque you can use a trackpad to scroll if it is set up to emulate the keyboard controls shown at the link above (eg Page up/Page down).

You can use folders in xwax, however we hide the folder browsing feature in the PiDeck build as it complicates use on small touchscreens. I'd suggest looking at genre tagging rather than folder navigation if you want to use a small screen without reworking the xwax GUI code.

**Comment by beaucasque** [comment]
Ok. I'm ordering a few missing pieces and will send you some pics of my new Pideck DJ booth.

Tagging is quite simple and a great way to index and group. You really have the way on using 100% of minimal ressources instead of 30% of too many.

Thank you again.

Patrice.
________________________________

**Comment by beaucasque** [comment]
Hi Daniel!

I’ve just installed Pideck on a 3B+. I’m using a Linux HD compatible touchscreen. Pideck takes only a part of the screen but it works. Touch and all. I could have changed the -g argument in the pideck file. but did’nt feel like recompiling awax as I’m a newbie. I prefer buying the official 7” pi display.

So here’s some mod that I did via SSH:

  1.  Modified the A6 name as it no named properly in sup_cards.conf. Also did all the modifications on the T6.asound.conf.
  2.  Changed the line 61 of the pideck file

From:     xwax -i $IMPORT -s $SCAN -k -q 89 -g 1920x1080 --no-decor $DICER -m $BUFFER -r $SAMPLERATE $XCARD $LIBRARY &

To:     xwax -i $IMPORT -s $SCAN -k -q 89 -g 1920x1080 --no-decor --dicer hw:Dicer -m $BUFFER -r $SAMPLERATE $XCARD $LIBRARY &
So at this point, It boots up properly but only shows the touchscreen_control left and right. Nothing in the middle. Only black.

I’ve tried with nothing connected and also tried with 1) dicers 2) Traktor Audio 6 3) Linux formatted FAT32 key 4) usb for my screen touch control (HID).

Thank you for you help.. Almost there!

Patrice

**Comment by danielhjames** [comment]
Hi @beaucasque if there is nothing in the middle of the touchscreen controls, it means xwax has failed to start. If you connect a USB keyboard and press any touchscreen control to get that program in focus, then F10 on the keyboard, a terminal should open. Then you can debug your xwax startup script without using ssh.

Once you have xwax running from the terminal, you can then make any changes required to the automatic startup scripts in /usr/share/pideck

Please let us know how you get on!

**Comment by beaucasque** [comment]
Ok. I'm now using a really official setup which is. Pi3B+ Traktor Audio 6 + Official 7" touch display.

My traktor Audio 6 has been updated so there might be a naming change from Traktor Audio 6 DJ to Traktor Audio 6 but for now, I will try to make xwax start. it's a fresh install. nothing modified in /usr/share/pideck

I pressed F10 and typed:
xwax -a plughw:0 -l ~/Music

I received this:
initialising deck 0 (hw:0)...
ALSA lib pcm_hw.c:1820:(_snd_pcm_hw_open) Invalid value for card
ALSA open: No such file or directory
Failed to open device for capture.

I had a FAT32 formatted drive with my music in a folder named Music on the root.

Thank you for your help!

**Comment by beaucasque** [comment]
Ok. I'm now using a really official setup which is. Pi3B+ Traktor Audio 6 + Official 7" touch display.

My traktor Audio 6 has been updated so there might be a naming change from Traktor Audio 6 DJ to Traktor Audio 6 but for now, I will try to make xwax start. it's a fresh install. nothing modified in /usr/share/pideck

I pressed F10 and typed:
xwax -a plughw:1 -l /media/usb0

it opened an xwax windows with only one player and even if it successfully scanned the usb mount, no music would show.

I had a FAT32 formatted drive with my music in a folder named Music on the root.

Thank you for your help!

**Comment by beaucasque** [comment]
@danielhjames I've posted in TraktorA6 issue as this is where I need to put my efforts. My setup is working. Please see my comment regarding this issue in the respective issue chat.
